service/worker/asyncworkflow/async_workflow_consumer_manager.go

```diff
@@ -175,17 +178,23 @@ func (c *ConsumerManager) refreshConsumers() {
 
 		c.logger.Info("Starting consumer", tag.WorkflowDomainName(domain.GetInfo().Name), tag.Dynamic("queue-id", queue.ID()))
 		consumer, err := queue.CreateConsumer(&provider.Params{
-			Logger:        c.logger,
-			MetricsClient: c.metricsClient,
+			Logger:         c.logger,
+			MetricsClient:  c.metricsClient,
+			FrontendClient: c.frontendClient,
 		})
 		if err != nil {
 			c.logger.Error("Failed to create consumer", tag.Error(err), tag.WorkflowDomainName(domain.GetInfo().Name), tag.Dynamic("queue-id", queue.ID()))
 			continue
 		}
 
+		if err := consumer.Start(); err != nil {
+			c.logger.Error("Failed to start consumer", tag.Error(err), tag.WorkflowDomainName(domain.GetInfo().Name), tag.Dynamic("queue-id", queue.ID()))
```

Comment:
nit: probably worth making a log tag for `queue-id`

Reply:
good point. I noticed I use it in multiple places now. let me turn that into a known tag